examples/guides/01-building-a-component-with-jco.md

```diff
@@ -0,0 +1,83 @@
+# 01 - Building a WebAssembly Component with `jco`
+
+The first step in building a WebAssembly component is creating or downloading the interface that defines what your component can do. This usually means creating or downloading the [WebAssembly Interface Types ("WIT")][wit] world you would like to "target" with your component.
+
+The [example `add` component in `jco`](https://github.com/bytecodealliance/jco/tree/main/examples/components/add) showcases a component that adds two numbers together. The WIT interface looks like the following:
+
+```wit
+package docs:adder@0.1.0;
+
+interface add {
+    add: func(a: u32, b: u32) -> u32;
+}
+
+world adder {
+    export add;
+}
+```
+
+> [!NOTE]
+> `export`ing the `add` interface meants that runners of the WebAssembly binary will be able to *call* that function.
+>
+> To learn more about the WIT syntax, check out the full [WIT specification][wit]
+
+Along with this WIT interface, we can write a JavaScript module that implements the exported `add` function in the `adder` world:
+
+```js
+export fucntion add(x, y) {
+  return x + y;
+}
+```
+
+> [!NOTE]
+> In the code above, the `add` interface is exported, and the `add` function *inside* that interface is implemented.
+
+With the WIT and Javascript in place, we can use [`jco`][jco] to create a WebAssembly component from the JS module, using `jco componentize`.
+
+Our component is *so simple* (reminiscent of [Core WebAssembly][wasm-core], which deals primarily in numeric values) that we're actually *not using* any of the [WebAssembly System Interface][wasi] -- this means that we can `--disable` it when we invoke `jco componentize`
+
+```console
+jco componentize \
+    --wit path/to/adder/world.wit \
+    --world-name adder \
+    --out add.wasm \
+    --disable all \
+    path/to/add.js
+```
+
+You should see output like the following:
+
+```
+OK Successfully written add.wasm.
+```
+
+> [!NOTE]
+> As the `add` example is a regular [NodeJS][nodejs] project, you can run `npm install && npm run build`
+> without having `jco` and `componentize-js` installed globally.
+
+[wit]: https://github.com/WebAssembly/component-model/blob/main/design/mvp/WIT.md
+[nodejs]: https://nodejs.org/en
+[wasi]: https://wasi.dev/
+[wasm-core]: https://webassembly.github.io/spec/core/
+
+## FAQ
+
+### Is this a Reactor component?
```

Comment:
Hey @pchickey would love a gutcheck here -- IIRC the difference between command and reactor is the `_start` export and semantics, is this section missing anything?